table format to list Links ? (#27)
minrk/horizon-widera-2022: concept.tex

  established; [e.g. 1 pages]}
\TOWRITE{}{}

> The SOURCE project, as part of the overall jupyter environment, will intrinsically build on the achievements of many national and European and international research and innovative developments, which have been produced or are being implemented in research projects in which research infrastructures and project partners are or have been involved.
> Table here \TODO{link table} shows a selection of highly relevant projects in different areas and coordinated and/or conducted by SOURCE partners.

Reviewer: ```suggestion
\TheProject, as part of the overall jupyter ecosystem,
will intrinsically build on the achievements of many national and European and international research and innovation activities,
which have been produced or are being implemented in research projects in which research infrastructures and project partners are or have been involved.
```
```suggestion
The SOURCE project, as part of the overall jupyter environment, will intrinsically build on the achievements of many national and European and international research and innovative developments, which have been produced or are being implemented in research projects in which research infrastructures and project partners are or have been involved.
Table~\ref{tab:national-research} shows a selection of highly relevant projects in different areas and coordinated and/or conducted by \TheProject partners.
```

\begin{table}[]

> \begin{tabular}{lll}

Reviewer: ```suggestion
\label{tab:national-research}
\begin{tabular}{lll}
```
